### ISSUE TITLE
How about changing the term "SDPX BOM" to "SPDX SBOM"?

### ISSUE THREAD

**Issue by shi9qiu** [eclipse/sw360#1353]
[//]: # (Copyright Siemens AG, 2021. Part of the SW360 Portal Project)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

### Description

Compared with "SDPX BOM" (term used in sw360 now), "SPDX SBOM" is more widely used and more reasonable. Shall we change to "SPDX SBOM"?

#### Screenshots

![spdx_name](https://user-images.githubusercontent.com/83331814/129687997-60b26adb-dad5-42d2-8b10-a89517c5c4d5.jpg)

### Versions

* Last commit id on master:
* Operating System (lsb_release -a):

### Logs

Any logs (if any) generated in

#### SW360 logs

Logs generated under /var/log/sw360/sw360.log

#### Tomcat logs

Logs generated under /var/log/tomcat/error.log

**Comment by mcjaeger** [comment]
I also agree on the change from "BOM" to "SBOM". I ahve heard some persons like to support more sbom formats, so how about changing it from "Import SPDX BOM" to "Import SBOM"?

**Comment by shi9qiu** [comment]
> I also agree on the change from "BOM" to "SBOM". I ahve heard some persons like to support more sbom formats, so how about changing it from "Import SPDX BOM" to "Import SBOM"?

Good idea. I think my team could update this together when we commit the SPDX related function next time.
@KoukiHama @quan-tv
